File: Java Script /Prova de Back-End 1°Tri /Maior número da lista.js
let vetor = [32, 9, 2, 45, 18];
let maior = 0;
for (let i = 0; vetor.length; i+1){
    if (vetor[i+1] > vetor[maior]){
        maior = i+1
    }
}

console.log("O maior número é", vetor[maior]);

File: Python /Maior Número da Lista.py
lista = []

# Inputs do Usuário
for i in range(1, 8):
    num = int(input("Digite um número: "))
    lista.append(num)

print("Número digitados:", lista)

# Função Principal
maior = 0
for i in range(len(lista)):
    if lista[i] > lista[maior]:
        maior = i
    else:
        continue

# Finalização
print("O maior número é", lista[maior])
